clients/client-py/README.md

```diff
@@ -9,45 +9,59 @@ This library is a complete interface to Taskcluster in Python.  It provides
 both synchronous and asynchronous interfaces for all Taskcluster API methods,
 in both Python-2 and Python-3 variants.
 
-## Usage
+## Authenticated Client Example
 
-For a general guide to using Taskcluster clients, see [Calling Taskcluster APIs](https://docs.taskcluster.net/docs/manual/using/api).
+For a general guide to using Taskcluster clients, see [Calling Taskcluster APIs](https://docs.taskcluster.net/docs/manual/using/api).     
+                                                                      
+#### Acquire Access Token for Client
```

Comment:
That does not work on Windows

Reply:
Hm, let's get those docs updated for how to make it work on Windows.  The tool does work on Windows, but I can imagine that the shell-style output doesn't.  If there's a way to do `eval $(..)` in DOS, let's figure that out, and otherwise maybe change the output to use `SET` and copy/paste.